qgreenland/util/qgis/project.py

```diff
@@ -87,6 +89,22 @@ def make_qgis_project_file(path: Path) -> None:
     project.clear()
 
 
+def _add_project_metadata(project: qgc.QgsProject) -> None:
+    """Add project-level metadata (title, authors, abstract).
+
+    This information is viewable in the **Project > Properties** in QGIS.
+    """
+    project_metadata = project.metadata()
+    project_metadata.setAuthor("QGreenland team")
```

Comment:
Should we use our names, or is "QGreenland team" appropriate here?

Reply:
I think `QGreenland team` is great. The full citation is easy to find. I think we should link to that page in our docs from this metadata.